ui/src/tasks/actions/index.ts

```diff
@@ -283,7 +308,7 @@ export const removeTaskLabelsAsync = (
 
 export const updateTaskStatus = (task: Task) => async dispatch => {
   try {
-    await client.tasks.updateStatus(task.id, task.status)
+    await patchTask({taskID: task.id, data: {status: task.status}})
```

Comment:
Since this only updates the status (and does not pass along any information relating the `cron`, `every` specifics), I did not run a check to see whether or not the cron / every props needed to be updated